scripts/create-native-package.sh

```diff
@@ -0,0 +1,67 @@
+## create package directory under /packages
```

Comment:
would it be simpler to start w/ running `create-react-native-library` and work from there?

Reply:
can this be removed now? I think all the info got moved to NATIVE_MODULE_DEVELOPING.md ?